### ISSUE TITLE
Aprovecha que tienes Tailwind CSS para aprender cómo funciona

### ISSUE THREAD

**Issue by sertxudev** [JosepJSaez/safraner#1]
😉

**Comment by JosepJSaez** [comment]
Yup, totalmente. Ahora que estoy con Vue3  + vite voy a aprovechar para aprender también Tailwind, siempre es bueno tener otra herramienta más, voy a montar un proyecto con todo ello para pillar soltura :P
